backend/controller/dal/internal/sql/queries.sql

```diff
@@ -205,29 +205,6 @@ FROM runners r
 WHERE state = 'assigned'
   AND d.key = sqlc.arg('key')::deployment_key;
 
--- name: InsertTimelineLogEvent :exec
-INSERT INTO timeline (
-  deployment_id,
-  request_id,
-  time_stamp,
-  custom_key_1,
-  type,
-  payload
-)
-VALUES (
-  (SELECT id FROM deployments d WHERE d.key = sqlc.arg('deployment_key')::deployment_key LIMIT 1),
-  (
-    CASE
-      WHEN sqlc.narg('request_key')::TEXT IS NULL THEN NULL
-      ELSE (SELECT id FROM requests ir WHERE ir.key = sqlc.narg('request_key')::TEXT LIMIT 1)
-    END
-  ),
-  sqlc.arg('time_stamp')::TIMESTAMPTZ,
-  sqlc.arg('level')::INT,
-  'log',
-  sqlc.arg('payload')
-);
-
 -- name: InsertTimelineDeploymentCreatedEvent :exec
```

Comment:
I wanted to remove these as well, but they are used in a transaction in the `controller/dal` and I didn't want to break that functionality for now. We could insert the timeline event after the transaction as well, which might be cleaner, but would slightly decouple the event from the action, just slightly though :)

Reply:
Definitely pull them out into a separate .sql file, and just add them to the controller/dal sql gen, like [here](https://github.com/TBD54566975/ftl/blob/main/sqlc.yaml#L5-L9).

Reply:
Ok I think I did this correctly, but please lemme know if I'm off the rails :)